pkg/sdk/logging/model/output/s3.go

```diff
@@ -242,7 +243,7 @@ func (c *S3OutputConfig) ToDirective(secretLoader secret.SecretLoader, id string
 		if c.Buffer == nil {
 			c.Buffer = new(Buffer)
 		}
-		c.Buffer.Tags = OneEyeTags
+		c.Buffer.Tags = &OneEyeTags
```

Comment:
I'd rather we use the `StringPointer` utility function from operator-tools to convert the `const` to a `*string`
```suggestion
		c.Buffer.Tags = utils.StringPointer(OneEyeTags)
```

Reply:
Great Thanks!